lib/message.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+
+class Message
+    def self.all
+        if ENV['ENVIRONMENT'] == 'test'
```

Comment:
This if/else statement is repeated below in the `.create` method as well. It's worth thinking about extracting this into its own method / helper.